scripts/validate-helm.sh

```diff
@@ -1,19 +1,17 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
 set -euo pipefail
-export KUBERAY_HOME=$(git rev-parse --show-toplevel)
-SCRIPT_PATH="${KUBERAY_HOME}/scripts/openapi2jsonschema.py"
-RAYCLUSTER_CRD_PATH="$KUBERAY_HOME/ray-operator/config/crd/bases/ray.io_rayclusters.yaml"
-tmp=$(mktemp -d)
-trap 'rm -rf "$tmp"' EXIT
 
-# Convert CRD YAML to JSON Schema
-pushd "${tmp}" > /dev/null
-"$SCRIPT_PATH" "$RAYCLUSTER_CRD_PATH"
-popd > /dev/null
-RAYCLUSTER_CRD_SCHEMA="${tmp}/raycluster_v1.json"
+raycluster_crd_schema=./schema/raycluster_v1.json
+
+if [ ! -f "$raycluster_crd_schema" ]; then
+  echo "CRD schema not found: $raycluster_crd_schema"
+  echo 'Please run "pre-commit genearete-crd-schema --all-files" first'
+  exit 1
+fi
+
+charts=("kuberay-apiserver" "kuberay-operator" "ray-cluster")
 
-# Validate Helm charts with kubeconform
-echo "Validating Helm Charts with kubeconform..."
-helm template "$KUBERAY_HOME/helm-chart/kuberay-apiserver" | kubeconform --summary -schema-location default
-helm template "$KUBERAY_HOME/helm-chart/kuberay-operator" | kubeconform --summary -schema-location default
-helm template "$KUBERAY_HOME/helm-chart/ray-cluster" | kubeconform --summary -schema-location default -schema-location "$RAYCLUSTER_CRD_SCHEMA"
+for chart in $charts; do
+  helm template ./helm-chart/$chart | kubeconform --summary -schema-location default -schema-location "$raycluster_crd_schema"
```

Comment:
nit: https://www.shellcheck.net/wiki/SC2086

```suggestion
  helm template ./helm-chart/"$chart" | kubeconform --summary -schema-location default -schema-location "$raycluster_crd_schema"
```